dataavailability/dataavailability.go

```diff
@@ -0,0 +1,34 @@
+package dataavailability
+
+import (
+	"context"
+
+	"github.com/0xPolygonHermez/zkevm-sequence-sender/etherman/types"
+)
+
+// DataAvailability implements an abstract data availability integration
+type DataAvailability struct {
+	backend DABackender
+}
+
+// New creates a DataAvailability instance
+func New(backend DABackender) (*DataAvailability, error) {
+	da := &DataAvailability{
+		backend: backend,
+	}
+	err := da.backend.Init()
+	return da, err
```

Comment:
```suggestion
	return da, da.backend.Init()
```

To follow the same style as in line 33

Reply:
Done